app/forms/admin/depositors_search_form.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Admin
+  # Admin form object for searching for depositor information
+  class DepositorsSearchForm < ApplicationForm
+    before_validation :normalize_druids
+    attribute :druids, :string
```

Comment:
Hmm ... I'm not sure about this being passed as a string but then transformed to an array. Perhaps they should be separate attributes (where the array attribute has `array: true`).

Reply:
@justinlittman I'd started with this as an array, but I've forgotten why I switched. Maybe because the supporting field in the HTML is a `textarea`? I'll take another hack at it. I prefer thinking of `druids` as an array, too.

Reply:
I'm going to skip this data type change for now, since the textarea component wants its field to be string-y.